yarn.lock

```diff
@@ -26355,7 +26290,7 @@ terser@5.14.2, terser@^5.3.4, terser@^5.5.1, terser@^5.7.2:
     source-map-support "~0.5.20"
 
 terser@^4.1.2, terser@^4.3.9, terser@^4.6.12, terser@^4.6.2, terser@^4.6.3:
-  version "4.8.1"
+  version "4.8.0"
```

Comment:
wondering what triggered this downgrade ?

Reply:
Not sure since the only dep change I made was to remove `chromedriver` and it does not depend on Terser. I've also pulled in upstream changes. Maybe something from there?